src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 20px;
+  background-color: black;
+}
+
+.block {
+  max-width: 350px;
+  margin-bottom: 40px;
+}
+
+.blue {
+  background-color: #308dc0;
+  border: 1px solid #e95d49;
+  padding: 10px;
+}
+
+.yellow {
+  background-color: #ffa500;
+  border: 1px solid #e95d49;
+  padding: 10px;
+}
+
+.text {
+  border: 1px solid #e95d49;
+  margin: 0;
+  padding: 10px;
+  background-color: #ddd;
+}
```

Comment:
You have same styles for border and padding in three selectors.
Remember, don't repeat yourself.
Create one more common class for these styles.

Also, you can safely combine `.blue` and `.block` into one class. They have no overlapping styles.